StringCalc/CalculatorHelpers.cs

```diff
@@ -0,0 +1,65 @@
+﻿namespace StringCalc
+{
+    public static class CalculatorHelpers
+    {
+        private static readonly string[] Seperators = { "\n", ","};
+        private static readonly string[] DelimiterStrings = { "//", "[", "][", "]" };
+        public enum Defaults
+        {
+            DefaultValue = 0,
+            MaxValue = 1000,
+        }
+        
+        public static bool ValuesAreNullOrEmpty(string numbers)
+        {
+            return string.IsNullOrEmpty(numbers);
+        }
```

Comment:
This is wrapping an existing, descriptive method in another method, which is providing no real benefit. May as well just call string.IsNullOrEmpty in the original place instead.